### PR TITLE
Desktop version of my portfolio

### PR DESCRIPTION
**Pull request by kelomo2502**
For the fourth project in my portfolio website:

- I use media queries to make sure that your website displays correctly on desktop screens.
- Make sure that I used correct ([Github flow])
- Follow the list of [best practices for HTML & CSS]

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -91,16 +98,15 @@ <h3>Multi-Post Stories</h3>
             <li><button type="button">Ruby</button></li>
           </ul>
         </div>
-        <button class="see-works cards-btn" type="button">See Project</button>
-      </div>
+        <button class="see-works cards-btn" type="button"  id="multipost-btn">See Project</button>
```
Comment (danielufeli) [inline index.html:101]: - Nice work you have done with your project to make sure everything looks good with the card kindly move the button to the bottom of the card, so it aligns with the Figma Template.

## Figma Design
card

## Your Design
card2

File: index.html [hunk 2]
```diff
@@ -196,7 +202,7 @@ <h4>About me</h4>
           like what you see and have a project you need coded, don’t hesitate to
           contact me.
         </p>
-        <button type="button" class="cards-btn">Get My Resume</button>
+        <button type="button" class="cards-btn" >Get My Resume</button>
       </div>
       <hr class="divider" />
       <div class="technologies">
```
Comment (danielufeli) [inline index.html:209]: - [ ] Nice work on the About Us section, kindly reduce the padding in between the `list` so it aligns with the Figma Design

## Your Design
sk-1

## Figma Design
sk-2

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @kelomo2502, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Highlights
Nice commits messages✔️
Descriptive PR ✔️
Clean Design ✔️
There is no linter error is present. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -196,7 +203,7 @@ <h4>About me</h4>
...
```
Comment (gtekle) [inline index.html:209]: - [ ] You should minimize the space between the list items to make it look like in the Figma design as the previous reviewer has suggested. You can try something like:
Removing  `**min-height: 50vh;**`  from above
```
.lang ul {
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
}
```
Adding the following to the **.technologies ul**
```
 .technologies ul {
    justify-content: flex-start;   
}
```

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @kelomo2502 :wave: , 

# Great work on making the changes requested by a previous reviewer 👏 :clap: :clap: 
You've done well implementing most of the requested changes, but there are still some which aren't addressed yet.
this-close


 ## Highlights
:heavy_check_mark: Splendid UI Design
:heavy_check_mark: Meaningful commit messages used
:heavy_check_mark: Good use of Github flow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kelomo2502** [comment]
Required changes implemented in accordance with the current reviewer

**Review by danielufeli — APPROVED** [review]
Hi @kelomo2502, 

Great work on making the changes requested by a previous reviewer,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Clean Design ✔️
- Good use of Github flow ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**